app/src/features/skjema-moduler/Inntekt.tsx

```diff
@@ -14,15 +14,13 @@ import { format } from "date-fns";
 import { nb } from "date-fns/locale";
 import { Fragment } from "react";
 
+import type { ForespørselDto } from "~/api/queries";
 import { InformasjonsseksjonMedKilde } from "~/features/skjema-moduler/PersonOgSelskapsInformasjonSeksjon.tsx";
-import type {
-  InntektsmeldingDialogDto,
-  MånedsinntektResponsDto,
-} from "~/types/api-models.ts";
+import type { MånedsinntektResponsDto } from "~/types/api-models.ts";
 import { capitalizeSetning, formatKroner, leggTilGenitiv } from "~/utils.ts";
 
 type InntektProps = {
-  inntektsmeldingDialogDto: InntektsmeldingDialogDto;
+  inntektsmeldingDialogDto: ForespørselDto;
```

Comment:
Usikker på hvilken terminologi vi skal legge oss på for den datapakka? Forespørsel blir litt feil synes jeg.

Noen forslag:
- InntektsmeldingGrunnlag
- Grunnlag
- InntektsmeldingOpplysninger
- ellerno
Tanker?

Reply:
Tja kan godt kalle det opplysninger kanskje? jeg kan endre jeg